### PR TITLE
vendor: remove getimports, move to using govendor

### PR DESCRIPTION
**Pull request by rminnich**
Govendor is what we wanted all along.

We now have a vendor tree and a clean way to extend it.
go get considered harmful and now ... it's gone.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

**Comment by rminnich** [comment]
I don't think it's worth it. Basically this is a change of not that many files, and then a vendoring of lots of stuff. If you ignore vendor/ it's not that large. I don't think adding each package one at a time adds real value.

**Comment by ryukinix** [comment]
Yes, I don't it's worth it too. The vendor is a old problem here, now if is gone using just a commit, I'm okay with it. Split this can be more confuse in git history later.

**Comment by rminnich** [comment]
Oops, I did not answer your questions. You can skip everything under vendor/. Other than that it's a remove and a few small changes.

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# For maintainers only
+
+# We use the github hub tool for code review. We use govendor for maintaining dependencies.
```
Comment (GanShun) [inline MAINTAINERS.md]: We don't do this anymore. Update the instructions?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Setup your u-root Github Repository
+
+Follow the instructions for using hub. At some point, you'll need to fork github.com/u-root/u-root, then
```
Comment (GanShun) [inline MAINTAINERS.md]: Likewise

**Pull request merged**